api/src/pcapi/routes/serialization/collective_offers_serialize.py

```diff
@@ -512,6 +518,12 @@ def validate_intervention_area_not_empty_when_specified(
 
         return intervention_area
 
+    @validator("bookingEmails")
+    def validate_booking_emails(cls, booking_emails: list[str]) -> list[str]:
+        if len(booking_emails) < 1:
```

Comment:
Style: pareil que le commentaire précédent
```suggestion
        if not booking_emails:
```